src/SAWScript/Crucible/Common.hs

```diff
@@ -89,14 +91,22 @@ newSAWCoreBackend pss sym = newSAWCoreBackendWithTimeout pss sym 0
 newSAWCoreBackendWithTimeout :: PathSatSolver -> Sym -> Integer -> IO SomeOnlineBackend
 newSAWCoreBackendWithTimeout PathSat_Yices sym timeout =
   do bak <- newOnlineBackend sym Yices.yicesDefaultFeatures
+     W4.extendConfig Z3.z3Options (W4.getConfiguration sym)
      W4.extendConfig Yices.yicesOptions (W4.getConfiguration sym)
+     W4.extendConfig CVC5.cvc5Options (W4.getConfiguration sym)
      yicesTimeoutSetting <- W4.getOptionSetting Yices.yicesGoalTimeout (W4.getConfiguration sym)
      _ <- W4.setOpt yicesTimeoutSetting timeout
+
+     auxOutSetting <- W4.getOptionSetting solverInteractionFile (W4.getConfiguration sym)
+     _ <- W4.setOpt auxOutSetting $ Text.pack "foo.smt2"
+
      return (SomeOnlineBackend (bak :: Backend Yices.Connection))
 
 newSAWCoreBackendWithTimeout PathSat_Z3 sym timeout =
   do bak <- newOnlineBackend sym (SMT2.defaultFeatures Z3.Z3)
      W4.extendConfig Z3.z3Options (W4.getConfiguration sym)
+     W4.extendConfig Yices.yicesOptions (W4.getConfiguration sym)
+     W4.extendConfig CVC5.cvc5Options (W4.getConfiguration sym)
```

Comment:
`addz3 addz3` is a failure in the second, but no addz3 is a failure.  This adds the backend-support for all solvers to allow flexibility in invoking them as alternatives during the analysis.